src/java/components/gpiomanager/src/test/java/hu/bme/mit/inf/modes3/components/gpiomanager/TestGpioWriter.java

```diff
@@ -0,0 +1,35 @@
+package hu.bme.mit.inf.modes3.components.gpiomanager;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class TestGpioWriter implements CommandWriter {
+
+	private static final String TAG = "GPIOTestWriter";
+
+	private Map<String, String> infos;
+
+	public TestGpioWriter() {
+		infos = new HashMap<String, String>();
+	}
+
+	@Override
+	public void executeCommand(String value, String targetFile) throws IOException {
```

Comment:
Is this method invoked by different `value` and `targetFile` parameters in the unit tests? If so, then does it that unit test test this class itself or it is just a side-effect that this method will also be invoked and executed when you test another method (of another class) which invokes this class? 

So, is there a reason why this 'test' class was implemented instead of just mocking it in the class/instance where it is invoked?

Reply:
Not really